src/components/Signature.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+
+type SignatureProps = {
+  text: string;
+};
+
+const popover = (
+  <div className="popover fade bs-popover-right show">
+    <div className="arrow" />
+    <div className="popover-body">Copied!</div>
+  </div>
+);
+
+function Signature({ text }: SignatureProps) {
```

Comment:
What do you think about making this a bit more generic and calling it `Copyable`?

1. Account addresses aren't signatures
2. We could use this for non `<code>` wrapped text too!

Also, I think it would be nice if this was more of a HOC. It'd be cool if it accepted `children` as well `text` for its props. So that way it could be used like so:

```jsx
<Copyable text={transaction.signature}>
  <code>{transaction.signature}</code>
</Copyable>
```

Reply:
Sounds good.  It occurred to me after the PR that addresses weren't signatures.  I like the added flexibility of accepting children.

Will change in the next 48 hours and re-submit.  Thanks for the feedback!